src/SamplesApp/Benchmarks.Shared/Suite/Windows_UI_Xaml_Controls/GridBench/SimpleGridBenchmark.cs

```diff
@@ -1,29 +1,69 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 using BenchmarkDotNet.Attributes;
 using Windows.Foundation;
+using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
+using BenchmarkDotNet.Engines;
 
 namespace SamplesApp.Benchmarks.Suite.Windows_UI_Xaml_Controls.GridBench
 {
+	[MinColumn, MaxColumn, MeanColumn, MedianColumn]
+	[SimpleJob(launchCount: 3, warmupCount: 10, targetCount: 30)]
```

Comment:
Don't change this, it's set globally somewhere else and it may impact the overall benchmark runtime.